tests/chainer_tests/test_variable.py

```diff
@@ -366,6 +366,67 @@ def test_addgrad_gpu_to_another_gpu(self):
             c = cp.full(3, 30, dtype=np.float32)
         self.check_addgrad(a, b, c)
 
+    def test_debug_print_cpu(self):
```

Comment:
`test_debug_print_cpu` and `test_debug_print_gpu` do not use attributes of `self`. So these need not be in `TestVariable` class. Could you consider the possibility of separate them as another class?